src/docker/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM kitware/trame
 
 RUN apt-get update && \
     apt-get install --no-install-recommends --yes \
-        libosmesa6-dev \
+    libosmesa6-dev \
+    git \
```

Comment:
I don't think you need git anymore

Reply:
Ok, removed in [46943b0](https://github.com/Kitware/pan3d/pull/25/commits/46943b000034de32534e682eeb8d2321665f9737)

Reply:
Should we use the new vtk-osmesa wheels to avoid libosmesa too?

ref https://discourse.vtk.org/t/status-update-vtk-python-wheels/11212